### DISCUSSION TITLE
Hack a Day feature

### DISCUSSION THREAD

**Discussion by MarkAlanJones** [robhagemans/hoard-of-bitfonts#18] [General]
https://hackaday.com/2023/02/04/reliving-a-bitmapped-past-with-a-veritable-hoard-of-bitmap-fonts

Note the comments link to yet more ROM bios fonts !!
https://int10h.org/oldschool-pc-fonts/fontlist/

**Comment by robhagemans** [comment]
Hey, that's cool!

I see it has a link to VileR's int10h page - it's a great page, I've had a
browse through it before. I should probably add a link to the README.
Slightly different focus than I have - making the fonts usable in unicode
rather than just documenting the originals, though I think he maintains a
page with a large number of raw dumps of VGA fonts as well. I haven't
really got to VGA fonts, there are just too many of them :)
